app/assets/javascripts/swagger-ui/lib/swagger-client.js

```diff
@@ -1154,11 +1154,18 @@ Operation.prototype.getMissingParams = function(args) {
 Operation.prototype.getBody = function(headers, args, opts) {
   var formParams = {}, body, key;
 
+
   for(var i = 0; i < this.parameters.length; i++) {
     var param = this.parameters[i];
     if(typeof args[param.name] !== 'undefined') {
       if (param.in === 'body') {
-        body = args[param.name];
+        if(typeof body !== 'undefined') {
+          let json = JSON.parse(body);
+          json[param.name] = args[param.name];
+          body = JSON.stringify(json);
+        } else {
+          body = args[param.name];
+        }
```

Comment:
Is this enabling multiple entries in `body`?

Reply:
Yes, and looking at the distributor docs this is exactly how our affiliate_id filter is working. 

```
Our distributor tools are designed to make it as easy as possible for distributors to integrate the HR Support Center into their affiliates’ websites and improve the resulting user experience.
```

Reply:
Before this, swagger was taking the affiliate_id and just replacing the user payload, which is why it was sending a malformed request body to the controller.